infrastructure/router/modules/aws-fargate/vpc.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_default_vpc" "default_vpc" {}
```

Comment:
I think this setup is unlikely in a real world. Let's make this configurable, so a user can deploy the router in an existing VPC.